chromadb/test/property/test_persist.py

```diff
@@ -172,11 +170,14 @@ class PersistEmbeddingsStateMachineStates(EmbeddingStateMachineStates):
     persist = "persist"
 
 
+MIN_STATE_CHANGES_BEFORE_PERSIST = 5
```

Comment:
changed from 10 -> 5

@HammadB if you think this will result in a sharp decrease in the quality of this test let me know

at 10 persist was almost never called

(also would probably be fixed, without decreasing this parameter, by restoring DeterministicRuleStrategy)

Reply:
Yeah makes sense